src/include/cpp-crypto/identities/address.hpp

```diff
@@ -10,6 +10,7 @@
 #ifndef ARK_IDENTITIES_ADDRESS_HPP
 #define ARK_IDENTITIES_ADDRESS_HPP
 
+#include <cstdbool>
```

Comment:
What uses cstdbool?

Reply:
https://github.com/ArkEcosystem/cpp-crypto/blob/7cd4503ce23ec0bddd2ca175c220f8335a97ab8a/src/include/cpp-crypto/identities/address.hpp#L47

Reply:
You do not need cstdbool to use bool in C++ code.
http://www.cplusplus.com/reference/cstdbool/

Reply:
Strange, could have sworn there was a platform where the bool header was required, but I don't seem to able to find it now.

I'll remove these.